### ISSUE TITLE
Tidy up FreeBayes

### ISSUE THREAD

**Issue by fnothaft** [bigdatagenomics/cannoli#67]
- [ ] bump to `quay.io/ucsc_cgl/freebayes:1.1.0--f7597a50b4449d4a963ace565a6ca7040ae10118`
- [ ] mount reference file into docker container
- [ ] support copying files to executor
- [ ] run in `-i` mode
- [x] ~specify `-region` when invoking FreeBayes~ not necessary
- [ ] VCF comes out mis-ordered
- [ ] use Avocado read/contig filtering mechanism (https://github.com/bigdatagenomics/avocado/issues/254)

**Comment by heuermh** [comment]
Can we include test resources from https://github.com/ekg/freebayes/tree/master/test/tiny here in `cli/src/test/resources`?  MIT License but the test data don't have a copyright notice to include.

**Comment by heuermh** [comment]
With the fix for https://github.com/bigdatagenomics/avocado/issues/254 (commit https://github.com/bigdatagenomics/avocado/commit/5f720cf5461b15bf5cc0d7ed476e704dec3634cc), do you intend that Cannoli would depend on Avocado?

**Comment by heuermh** [comment]
Taking a closer look at the freebayes command line arguments, we should always use
```
   --strict-vcf
                   Generate strict VCF format (FORMAT/GQ will be an int)
```

and should provide arguments in the Freebayes wrapper for
```
   -A --cnv-map FILE
                   Read a copy number map from the BED file FILE, which has
                   the format:
                      reference sequence, start, end, sample name, copy number
                   ... for each region in each sample which does not have the
                   default copy number as set by --ploidy.

   --gvcf
                   Write gVCF output, which indicates coverage in uncalled regions.

   --gvcf-chunk NUM
                   When writing gVCF output emit a record for every NUM bases.
```

As to the rest, I'm not sure if we should wrap all of them, or provide a bulk argument/config mechanism, or simply say if you need to provide non-default parameters, call `pipe` manually.

**Comment by heuermh** [comment]
Fixed by #107.
